docs/specs/xref.yml

```diff
@@ -152,23 +152,30 @@ outputs:
   xrefmap.json: | 
     {"references":[{"uid":"a","href":"docs/a.json"}]}
 ---
-# Define xref property without uid at root level, should output warning
+# Define uid with error
 inputs:
   docfx.yml: |
     buildInternalXrefMap: true
   docs/a.json: |
     {
       "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
-      "uid": "",
+      "uid": "a",
       "name": "test",
       "fullName": "test.test",
-      "description": "a.description",
-      "data": {"name": "name without uid"}
+      "description":
+    }
+  docs/b.json: | 
+    {
+      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+      "xref": "a"
     }
 outputs:
-  docs/a.json:
+  docs/b.json: | 
+    {
+      "content": {}
```

Comment:
assert content does not have xref property using `{ "content": { "!xref": null } }`